docs/how-to-guides/running-feast-in-production.md

```diff
@@ -144,3 +185,142 @@ prediction = model.predict(feature_vector)
 It is important to note that both the training pipeline and model serving service only needs read access to the feature registry and associated infrastructure. This prevents clients from accidentally making changes to the feature store.
 {% endhint %}
 
+## 4. Retrieving online features for prediction
+
+Once you successfully loaded (or in Feast terminology materialized) your data from Batch Source into Online Store you can start consuming features for inference phase.
+There are three approaches for that purpose sorted from the most simple one (in operational sense) to the most performant:
```

Comment:
maybe call out that we'll be publishing benchmarks to help inform which option makes the most sense?

@tsotnet fyi